tutorials/Additive_and_multiplicative_bias.ipynb

```diff
@@ -964,16 +968,16 @@
     "import matplotlib.pyplot as plt\n",
     "fcst_flow = fcst.temp_scrn\n",
     "obs_flow = obs.temp_scrn\n",
-    "# Calculate additive bias precentage and preserve the \"lat\" dimension\n",
+    "# Calculate percent bias and preserve the \"lat\" dimension\n",
     "bias = pbias(fcst_flow, obs_flow, preserve_dims=\"lat\")\n",
     "bias.name = \"percent bias (%)\"\n",
     "# Generate second dataset with large flow values\n",
     "fcst_flow_large=fcst_flow + 1000\n",
     "obs_flow_large = obs_flow + 1000\n",
-    "# Calculate additive bias precentage for large flow values and preserve the \"lat\" dimension\n",
+    "# Calculate percent bias for large flow values and preserve the \"lat\" dimension\n",
```

Comment:
@durgals - as above, was it correct to make this change (to percent bias)?

Reply:
@Steph-Chong : Yes.